src/bitmap/iter.rs

```diff
@@ -59,6 +80,41 @@ impl<'a> Drop for BitmapIterator<'a> {
     }
 }
 
+pub struct BitmapChunks<'a> {
+    iterator: BitmapIterator<'a>,
+    size: usize,
+}
+
+impl<'a> BitmapChunks<'a> {
+    pub fn new(iterator: BitmapIterator<'a>, size: usize) -> Self {
+        BitmapChunks { iterator, size }
+    }
+}
+
+impl<'a> Iterator for BitmapChunks<'a> {
+    type Item = Vec<u32>;
+
+    fn next(&mut self) -> Option<Self::Item> {
+        let mut buffer = Vec::with_capacity(self.size);
+
+        unsafe {
+            let returned_chunk_size = ffi::roaring_read_uint32_iterator(
+                self.iterator.iterator,
+                buffer.as_mut_ptr(),
+                self.size.try_into().unwrap()
+            );
+
+            buffer.set_len(returned_chunk_size.try_into().unwrap());
```

Comment:
Is this conversion needed? set_len works with `usize`, which is what this ffi binding returns, right?

Reply:
Not really, the [ffi binding returns `u32`](https://docs.rs/croaring-sys/0.4.0/croaring_sys/fn.roaring_read_uint32_iterator.html), so I have to leave the conversion.